phpstan.core.neon

```diff
@@ -0,0 +1,7 @@
+includes:
+  - %currentWorkingDirectory%/vendor/centreon/centreon-test-lib/src/PHPStan/phpstan.neon
```

Comment:
according to [documentation](https://phpstan.org/config-reference#multiple-files) : `Relative paths in the includes section are resolved based on the directory of the config file is in`
```suggestion
  - vendor/centreon/centreon-test-lib/src/PHPStan/phpstan.neon
```

Reply:
Corrected include param